.gitignore

```diff
@@ -37,3 +44,15 @@ tests/.ipynb_checkpoints/
 
 # Ignore VCS files
 .idea/*
+
+# Ignore class directory
+class/
+Mead/
```

Comment:
I imagine there's a bunch of these things that won't be there in the end

Reply:
Yes, what is the general way of dealing with this type of thing? Basically I want a .gitignore that applies to the master CCL and that I don't mess around with, but then I also want another .gitignore that avoids any extra files or folders I may be messing around with in the main CCL directory. Or should I just not be adding files or folders here at all?

Reply:
Done, although I kept the .gitignore for the class directory. Is that okay?